frontend/src/pages/Screening.vue

```diff
@@ -1,40 +1,466 @@
+                    // { 'StudentID': 31232, 'FullName': 'Tina2', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID':  3, 'PhoneNumber': '81234102', 'EnglishProficiency':null, 'NativeLanguageID':4},                    
+                    // { 'StudentID': 41231, 'FullName': 'Clarence', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID': 4,  'PhoneNumber': '93141234','EnglishProficiency':'Little', 'NativeLanguageID':1 },
+                    // { 'StudentID': 53212, 'FullName': 'Anne', 'created_at': '2020-10-27T06:18:24.738Z', 'StatusID': 1,  'PhoneNumber': '81230532','EnglishProficiency':'Intermediate', 'NativeLanguageID':2 },
+
+                 ],
+
+                selected: null,
+                sortIcon: 'arrow-up',
+                sortIconSize: 'is-small',
+                sortDirection: 'asc',
+                checkedRows:[],
+                checkboxPosition: 'right',
+                uniqueEnglishVals: ['No', 'Little', 'Limited', 'Intermediate'],
+                isActive: false,
+                columns: [
+                    {
+                        field: 'created_at',
+                        label: 'Date Joined',
+                        searchable: true,
+                    },
+                    {
+                        field: 'StudentID',
+                        label: 'Student ID',
+                        searchable: true,
+                    },
+                    {
+                        field: 'FullName',
+                        label: 'Student Name',
+                        searchable: true,
+                    },    
+                    {
+                        field: 'NativeLanguageID',
+                        label: 'Native Language',
+                        searchable: true,
+                    },      
+                    {
+                        field: 'StatusID',
+                        label: 'Status',
+                        searchable: true,
+                    },
+                    {
+                        field: 'PhoneNumber',
+                        label: 'Phone Number',
+                        searchable: true,
+                    },
+                    {
+                        field: 'EnglishProficiency',
+                        label: 'English Proficiency',
+                        searchable: true,
+                    },
+                    {
+                        field: 'Dummy',
+                        label: 'Ready to Match:',
+                        searchable: true,
+                    },
+
+
+                ]
+            }
+        },
+
+
+        computed: {
+          tableData(){
+            return this.data.map(student=>{
+
+              let NativeLanguageDesc = "";
+              let SetNullEnglish ="";
+
+              if (student.NativeLanguageID === 1){
```

Comment:
Remember, we should not make any assumptions about what the ID is. When we deploy the final version of the database for real use, there is no guarantee that `"Bangla"` will have an ID of `1`. It would be better if the backend helps us out here by returning the JOINed data (like what we discussed during the planning), so that we get the description back as part of `this.data`.

Reply:
Yeah, hard-coded just to see it all come together on the front-end. The backend PR that addresses this is #42 right, will review it once the conflicts in #42 are resolved

Reply:
Update: Awaiting modification to include native language text when requesting for list of students